ide/static/js/content.js

```diff
@@ -230,6 +230,7 @@ class Content extends React.Component {
             downloadAnchor.download = response.name;
             downloadAnchor.href = response.url;
             downloadAnchor.click();
+            this.initialiseImportedNet(this.state.net,this.state.net_name);
```

Comment:
model initialises itself with the last state before exporting after every successful export.